### PR TITLE
Fix doc typo and VDR reported issues

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Fix typos
- For github links change to use github_nvflare_link

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: docs/examples/tutorial_notebooks.rst [hunk 1]
```diff
@@ -4,6 +4,6 @@ Tutorial Notebooks
 
 .. toctree::
 
-   FL Simulator Notebook (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_simulator.ipynb>
-   Hello FLARE API Notbook (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_api.ipynb>
-   NVFLARE in POC Mode (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/setup_poc.ipynb>
+   :github_nvflare_link:`FL Simulator Notebook (GitHub) <examples/tutorials/flare_simulator.ipynb>`
+   :github_nvflare_link:`Hello FLARE API Notebook (GitHub) <examples/tutorials/flare_api.ipynb>`
+   :github_nvflare_link:`NVFLARE POC Mode in details Notebook (GitHub) <examples/tutorials/setup_poc.ipynb>`
```
Comment (SYangster) [inline docs/examples/tutorial_notebooks.rst:9]: minor thing, "in detail"

Comment (YuanTingHsieh) [reply docs/examples/tutorial_notebooks.rst:9]: Thanks good catch will fix

**Review by SYangster — APPROVED** [review]
LGTM

**Pull request merged**